### PR TITLE
adds a what is Innersource section to the home page

### PR DESCRIPTION
**Pull request by andrea-kyurchiev**
Adds a section explaining what Innersource is to the home page. Currently looks like this:
image

The image could be replaced with any image from: https://storyset.com/people using color code 0AA8A7

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -29,8 +29,23 @@ title: "InnerSource Commons"
...
 
-
 <section class="section bg-light">
+  <div class="container">
+    <div class="row text-center">
+      <div class="col-md-5 my-auto">
+        <img src="/images/community/problem-solving.png" class="img-fluid pl-4 pr-4">
+      </div>
+      <div class="col-md-6 text-right">
+        <p class="section-title h2">What Is InnerSource?</p>
```
Comment (yuhattor) [inline content/en/_index.md]:
```suggestion
        <p class="section-title h2">What is InnerSource?</p>
```
"is" is more natural to me. but need native check

**Comment by yuhattor** [comment]
Overall it looks good to me!

**Review by rrrutledge — APPROVED** [review]
Looks great!

**Review by voborgus — APPROVED** [review]
great work!

**Pull request merged**